src/resources/elements/horizontal-scroller/horizontal-scroller.scss

```diff
@@ -8,7 +8,7 @@
     display: flex;
     flex-wrap: nowrap;
     overflow-x: auto;
-    width: calc(100% - calc(100vw - max-content));
+    width: 100%;
```

Comment:
have you checked whether this breaks the other places where it is used?  (Though I gotta say, the original line looks suspect)

Reply:
Yeah I checked the home page and the all deals page. There was a problem with the original line where it wasn't scrolling properly.